### PR TITLE
Fixed #25468 -- Made DjangoJSONEncoder lazy strings aware

### PR DESCRIPTION
**Pull request by claudep**
Thanks Stavros Korokithakis for the report.

### CODE REVIEW

File: docs/topics/serialization.txt [hunk 1]
```diff
@@ -256,16 +256,16 @@ Date and datetime related types are treated in a special way by the JSON
 serializer to make the format compatible with `ECMA-262`_.
 
 Be aware that not all Django output can be passed unmodified to :mod:`json`.
-In particular, :ref:`lazy translation objects <lazy-translations>` need a
-`special encoder`_ written for them. Something like this will work::
+For example, if you have some custom type in an object to be serialized, you'll
+have to write a `special encoder`_ written for it. Something like this will
```
Comment (timgraham) [inline docs/topics/serialization.txt]: chop "written"

**Comment by timgraham** [comment]
Worth a mention in the release notes? In commit message: "lazy string aware".

**Comment by claudep** [comment]
1.9 or 1.10?

**Comment by timgraham** [comment]
I think 1.10 since it seems to be more of a feature than a bug.

File: docs/releases/1.10.txt [hunk 2]
```diff
@@ -162,6 +162,12 @@ Requests and Responses
 
 * ...
 
+Serialization
+^^^^^^^^^^^^^
+
+* The ``django.core.serializers.json.DjangoJSONEncoder`` now knows how to
+  serialize lazy strings typically used for translatable content.
```
Comment (timgraham) [inline docs/releases/1.10.txt]: might add a comma after "strings", otherwise LGTM.

**Pull request merged**